src/views/frontend/widget/campaign/search_advisor.tpl

```diff
@@ -0,0 +1,2 @@
+<ff-campaign-advisor unresolved>
```

Comment:
I think adding only `ff-campaign-advisor` will not bring any visual effect. It has to be implemented together with
`ff-campaign-advisor-question` and `ff-campaign-advisor-answer`
https://web-components.fact-finder.de/api/4.x/ff-campaign

Reply:
Well. Actually in my tests that was not necessary. Those elements were added by the web-components module on its own, as it seems. I got a working search result page with just that.

Reply:
Hey,
It's rather unexpected for these elements to appear without defining their templates. but i will check this out.
thanks for letting me know!

Reply:
I was actually expecting that, because it worked the same way for other campaigns before. IIRC the "unresolved" was important for this to work. In many cases there seem to be default templates and only if you want to adapt that (e.g. put the questions and answers into separate containers) you need to provide a more detailed "template"

Reply:
You are right, the default templates are used when no custom one is provided, but when it comes to campaigns, the things are a little bit more difficult. The campaigns allows user to define HTML so in order to render it correctly we need to specify a template with  **{{{triple mustache syntax}}}.**. A default one uses regular double curly braces ( {{text}}) which causes html to be rendered in escaped form.
![image](https://user-images.githubusercontent.com/4511883/121312258-7fbc5300-c905-11eb-91b0-09548043c010.png)

It looks like our documentation is a little bit outdated because the suggested template also uses only double curly braces. We will likely to update the article.
If you don't mind i would ask you to add a simple template for the question and answers. It just a couple of lines but could saves us from situation where a customer complains that his campaign containing HTML is not rendered correctly

The `unresolved` attribute should be used for each element which renders a HTML to prevent rendering in in a not transformed form.

Reply:
Like this?

Reply:
yes, i was thinking about that.